web-app/src/main/java/com/google/sps/data/QuadTree.java

```diff
@@ -3,9 +3,6 @@
 import java.util.ArrayDeque;
 import java.util.ArrayList;
 import java.util.Deque;
-import com.google.gson.Gson;
-import com.google.sps.data.PoliceReport;
```

Comment:
Out of curiosity why are you deleting all these imports??

Reply:
We didn't use them!